modules/ask_va_api/app/lib/ask_va_api/optionset/retriever.rb

```diff
@@ -7,26 +7,8 @@ class OptionsetRetrieverError < StandardError; end
     class Retriever < BaseRetriever
       attr_reader :name
```

Comment:
Can `name` be removed too? It looks like it's also referenced in modules/ask_va_api/spec/app/lib/ask_va_api/optionset/retriever_spec.rb on line 11

Reply:
Good catch. Updated